atlascharts/split-boxplot.js

```diff
@@ -0,0 +1,252 @@
+/* 
+
+Copyright 2017 Observational Health Data Sciences and Informatics
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+Authors: Christopher Knoll
+
+*/
+
+define(["d3", "./chart"],
+	function(d3, Chart) {
+	"use strict";
+
+	class SplitBoxplot extends Chart {
+	  render(data, target, w, h, chartOptions) {
+	    // options
+			const defaults = {boxHeight: 10};
+			
+	    const options = this.getOptions(defaults, chartOptions);
+	    // container
+	    const svg = this.createSvg(target, w, h);
+
+	    const valueFormatter = this.formatters.formatSI(3);
+
+	    this.useTip((tip) => {
```

Comment:
Seems to fully repeat the tip in `boxplot.js`. Cannot we move it some utility file?